examples/python-guide/advanced_example.py

```diff
@@ -210,6 +210,7 @@ def callback(env):
         if env.iteration - env.begin_iteration == 5:
             print('Add a new valid dataset at iteration 5...')
             env.model.add_valid(lgb_eval_new, 'new_valid')
+
```

Comment:
can you please remove this unrelated whitespace change?

Reply:
ok， when I use  `Reformat code` command in pycharm, the new blank line will add. Anyway  I have remove it.